extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.css

```diff
@@ -36,8 +39,123 @@ amp-story-subscriptions.i-amphtml-story-subscriptions-visible::before {
   transition: opacity 0.2s cubic-bezier(0, 0, 0.2, 1) !important;
 }
 
-/** Make the dialog wrapper from amp-subscriptions round corner. */
+/** Override the dialog styles from amp-subscriptions. */
 amp-subscriptions-dialog.i-amphtml-story-subscriptions-dialog-wrapper {
-  border-radius: 16px 16px 0px 0px !important;
+  border-radius: 16px 16px 0 0 !important;
   font-family: Poppins, sans-serif !important;
+  font-weight: 400 !important;
+  background: white !important;
+  color: black !important;
+  overflow: hidden !important;
+}
+amp-story:not([supports-landscape]) + amp-subscriptions-dialog.i-amphtml-story-subscriptions-dialog-wrapper {
+  width: 100% !important;
+  left: 0 !important;
+  margin-left: 0 !important;
+}
+@media(min-aspect-ratio: 3/4) {
+  amp-story:not([supports-landscape]) + amp-subscriptions-dialog.i-amphtml-story-subscriptions-dialog-wrapper {
+    width: var(--i-amphtml-story-desktop-one-panel-width) !important;
+    left: 50% !important;
+    margin-left: calc(-0.5 * var(--i-amphtml-story-desktop-one-panel-width)) !important;
+  }
+}
+
+.i-amphtml-story-subscriptions-dialog-banner {
+  height: 2.54em !important;
+  background: #EDEDED !important;
+}
+
+.i-amphtml-story-subscriptions-dialog-content {
+  margin: 2.1em 1.6em 0 !important;
+  text-align: center !important;
+  white-space: nowrap !important;
+}
+
+.i-amphtml-story-subscriptions-price {
+  font-weight: 400 !important;
+  font-size: 1.4em !important;
+  letter-spacing: -0.02em !important;
+  text-overflow: ellipsis !important;
+  overflow: hidden !important;
+  margin-bottom: 0.3em !important;
+}
+
+.i-amphtml-story-subscriptions-title {
```

Comment:
When this isn't rendered there is a smaller margin between the `i-amphtml-story-subscriptions-price` and `i-amphtml-story-subscriptions-subtitle-first`. This needs some of the top margin to make that work.

<img width="600" alt="Screen Shot 2022-04-13 at 3 08 47 PM" src="https://user-images.githubusercontent.com/3860311/163261929-b6d6a446-a9a4-46c8-a450-b8d3d3522c8c.png">